opensrp-chw-core/src/main/java/org/smartregister/chw/core/utils/VisitVaccineUtil.java

```diff
@@ -53,11 +53,13 @@
 
             List<VaccineRepo.Vaccine> allVacs = new ArrayList<>();
 
-            List<VaccineRepo.Vaccine> woman =   ImmunizationLibrary.getVaccineCacheMap().get("woman").vaccineRepo;
-            allVacs.addAll(woman);
+            if (ImmunizationLibrary.getVaccineCacheMap().containsKey("woman")) {
+                allVacs.addAll(ImmunizationLibrary.getVaccineCacheMap().get("woman").vaccineRepo);
+            }
 
-            List<VaccineRepo.Vaccine> child =   ImmunizationLibrary.getVaccineCacheMap().get("child").vaccineRepo;
-            allVacs.addAll(child);
+            if (ImmunizationLibrary.getVaccineCacheMap().containsKey("child")) {
```

Comment:
Could we use the Vaccine Type constants [in the class here](https://github.com/OpenSRP/opensrp-client-immunization/blob/a194fdb008934623652bd0845930f67e750a59d6/opensrp-immunization/src/main/java/org/smartregister/immunization/util/IMConstants.java#L25)